lib/rules/not-declared.ts

```diff
@@ -110,6 +110,9 @@ export class RNotDeclared extends RuleBase implements IRule {
       if (obj instanceof OInstantiation) { // Instantiation handled somewhere else, where?
         continue;
       }
+      if (obj instanceof OLibraryReference) { // handled somewhere else
+        continue;
+      }
```

Comment:
Maybe add where it is handled :D

Reply:
Oh, I just assumed it was maybe handled somewhere else. No idea if it actually is handled ;)